llvm/lib/Target/Mips/MipsSEISelLowering.cpp

```diff
@@ -1244,6 +1244,12 @@ SDValue MipsSETargetLowering::lowerBITCAST(SDValue Op,
 
   // Bitcast double to i64.
   if (Src == MVT::f64 && Dest == MVT::i64) {
+    // Skip lower bitcast when operand0 has converted float results to integer
+    // which was done by function SoftenFloatResult.
+    if (DAG.getTargetLoweringInfo().getTypeAction(
+            *DAG.getContext(), Op.getOperand(0).getValueType()) ==
+        TargetLowering::TypeSoftenFloat)
```

Comment:
DAG.getTargetLoweringInfo() is this

Reply:
> DAG.getTargetLoweringInfo() is this

Sorry, I did not understand your comment.

Reply:
This method is part of MipsSETargetLowering so its part of what DAG.getTargetLoweringInfo() is pointing to (ie. the this pointer) - you can just use:
```cpp
if (getTypeAction(*DAG.getContext(), Op.getOperand(0).getValueType()) == TargetLowering::TypeSoftenFloat)
```

Reply:
> This method is part of MipsSETargetLowering so its part of what DAG.getTargetLoweringInfo() is pointing to (ie. the this pointer) - you can just use:
> 
> ```c++
> if (getTypeAction(*DAG.getContext(), Op.getOperand(0).getValueType()) == TargetLowering::TypeSoftenFloat)
> ```

Oh, I got it and applied it, thanks!